unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1657,7 +1658,7 @@ notifyUser dir o = case o of
     _nameChange _cmd _pastTenseCmd _oldName _newName _r = error "todo"
     expectedNonEmptyPushDest writeRemotePath =
       P.lines
-        [ P.wrap ("The remote namespace" <> prettyWriteRemotePath writeRemotePath <> "is empty."),
+        [ P.wrap ("The remote namespace " <> prettyWriteRemotePath writeRemotePath <> " is empty."),
```

Comment:
I think that due to the way `P.wrap` works that this change doesn't actually affect the output.

Reply:
I'd added the spaces before a merge which added the wrap 🙃